TAO/tao/tao.mpc

```diff
@@ -355,6 +355,7 @@ project(TAO) : acelib, install, tao_output, taodefaults, pidl, extra_core, taoid
     Basic_Arguments.h
     Basic_Argument_T.h
     Basic_Types.h
+    Basic_Types_IDLv4.h
```

Comment:
Do you not test against your installation set?

Reply:
There is currently no sponsor for doing that, feel free to add a github to test that

Reply:
Moved that change to its own PR, see https://github.com/DOCGroup/ACE_TAO/pull/1590